fix(docs-infra): don't wrap subcommands in code element

alan-agius4 opened the pull request:
With this change we no longer wrap subcommands titles and sub navigation items in code element for nicer UI

**Before**
![Screenshot 2022-03-14 at 13 35 24](https://user-images.githubusercontent.com/17563226/158174320-390ee98e-6fcd-4bbf-a3e2-97823da1c86f.png)

![Screenshot 2022-03-14 at 13 33 57](https://user-images.githubusercontent.com/17563226/158174329-5c06260c-efb4-45fd-8692-f0beab36433d.png)

**After**
![Screenshot 2022-03-14 at 13 43 14](https://user-images.githubusercontent.com/17563226/158174789-ea56dce2-fe11-4a98-aaae-17f819497b5f.png)

![Screenshot 2022-03-14 at 13 40 43](https://user-images.githubusercontent.com/17563226/158174926-2049842d-d6ec-4c56-bcf6-ec54c82e3ed9.png)

gkalpak reviewed (approved):
Nit: `docs-infra` would be a better commit message scope for this commit.

alan-agius4 commented:
@gkalpak, oops my mistake, you are right. I’ll fix that tomorrow.

alan-agius4 committed:
fix(docs-infra): don't wrap subcommands in code element

With this change we no longer wrap subcommands titles and sub navigation items in code element for nicer UI

mary-poppins commented:
You can preview f0a7199 at https://pr45343-f0a7199.ngbuilds.io/.

alxhub commented:
This PR was merged into the repository by commit aec9abd3747bd856f87572fdf7f85cc88a4f64ad.
